pkg/apis/hco/v1alpha1/hyperconverged_types.go

```diff
@@ -14,6 +14,16 @@ import (
 // HyperConvergedName is the name of the HyperConverged resource that will be reconciled
 const HyperConvergedName = "kubevirt-hyperconverged"
 
+// KubevirtConfiguration are the default configurations to be passed to kubevirt-configMap
+type KubevirtConfiguration struct {
+	DefaultNetworkInterface string `json:"default-network-interface,omitempty"`
```

Comment:
JSON naming should be camelCase. Please change to:

```golang
DefaultNetworkInterface string `json:"defaultNetworkInterface,omitempty"`
```

Reply:
@nunnatsa 
I understand it is a strong convention. I made it this way (with the `-`), so that it will match the keys in kubevirt-configMap. 
https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-config/config-map.go#L52
I am not sure it is a good enough reason.
what do you think?

Reply:
> what do you think?

it should be json complaint

Reply:
Done.